### ISSUE TITLE
Make sure all updates to smogon/usage-stats are reflected in @pkmn/stats

### ISSUE THREAD

**Issue by scheibo** [pkmn/stats#13]
`@pkmn/stats` is based on version of https://github.com/Antar1011/Smogon-Usage-Stats that was used by Smogon around ~2020, though which is out of sync with https://github.com/smogon/usage-stats. This issue will remain open until #10 is closed and will track changes made to stay in sync (modulo work such as #12 which is already a known feature gap), include for any changes made to the upstream repo after this point.

**Comment by scheibo** [comment]
1. `RunMonthly.sh` seems to replace `Catchup.sh`, but #11 is going to basically manage `RunMonthly.sh`'s job anyway
2. `MonotypeAnalysis.sh` now uses 5 processes instead of 3 but otherwise has just been updated from `gen7mono` -> `gen9mono`
3. `MonthlyAnalysis.sh` has had its tiers updated, upped to 5 processes *and* split out the 1695 and other stats such that they can all process in parallel. Once again, #11 should simply look at the upstream versions of what its trying to replace as it will cover this
4. `RunPartial.sh`/`PartialAnalysis.sh` mostly just seem to exist to allow for running over a small range of days (eg. to quickly figure out how a meta has progressed or for DLC etc). this seems like something also easily covered by #11 - really, stats should get processed and compressed daily and then running reporting ranges can just some up the various days
5. All the `TierUpdate*.py` files should be covered by #12
6. `common.py`:
	- now uses a predefined formats-data list instead of fetching from Github
	- aliases are expanded for new forms
	- ~~`nonSinglesFormats`/`non6v6Formats` are updated with several modern formats (note #9)~~
7. `TA.py`: basically still unchanged (maybe indicates no one cares and it should be deleted...), `StatCounter.py` is also unchanged
8. `batchLogReader.py`:
	  - ~has tera support~ this should have been covered by #1 
	  - !handles zacian/zazamenta~ (already covered in e63dff9386280bb2c27f52b253bea05aa6149c6e?)
	  - ~changes tiers where mega ray is allowed~
9. `batchMovesetCounter.py`
  	- ~has tera support~ ( #1)
    - ~teammate stats have changed, as has the off-by-1 in reporting them~ (fixed by 2d54ea26c7b0d8922d8d8a19b566a3fa053bbc88?)

**Comment by scheibo** [comment]
Closing this out because everuthing should be tracked elsewhere. Can reopen if upstream makes changes
